doc/configuration.md

```diff
@@ -105,14 +105,27 @@ Disables fetching pool offchain metadata.
 
 Disables all data related to governance
 
-### --disable-all
+### --disable-all : Experimental
 
 Disables almost all data except `block`, `tx` and data related to the ledger state
 
+### --dont-use-ledger : Experimental
+
+Maintains the ledger state, but doesn't use any of its data, except to load UTxO. To be used with `--bootstrap-tx-out`
```

Comment:
Should there be a flag combination that would skip maintaining ledger-state until reaching tip? Currently, it still takes ledger-state dump periodically during the sync. I'd have thought `--bootstrap-tx-out` would only maintain ledger-state upon block/tx table has synched to tip?

Reply:
> Should there be a flag combination that would skip maintaining ledger-state until reaching tip (and then execute once)?

This can be done, but I'm not sure if it's worth it. The only benefit I can think of is that ledger state and the memory spike would be maintained for a smaller duration. After db-sync reaches tip without the ledger state it would have to restart the chainsync protocol from genesis to get again all the blocks and apply the ledger rules. Also i don't understand the `execute once` part.

> Currently, it still takes ledger-state dump periodically during the sync

This is an orthogonal improvement we can make, which applies to every long syncing scenario: we shouldn't take that frequent or any snapshots.

> Add entry in a config table to denote migration was done (to ignore bootstrap-tx-out even if specified)

This one exists in table `extra_migrations`

Reply:
> After db-sync reaches tip without the ledger state it would have to restart the chainsync protocol from genesis to get again all the blocks and apply the ledger rules.

Oh - I misunderstood then, I thought similar to CLI/Ogmios, DBSync could fetch 'current' ledger-state dump from node without 'having' to validate entire history

Reply:
I don't think these state queries are very reliable and with UTxO-HD coming, the available queries may change. Ideally the node would provide a way to stream UTxO from its current state, which would result in no memory spike on the DBSync side.

We will sync with the consensus team on this manner.